src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,35 @@
+package roomescape.controller;
+
+import jakarta.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import roomescape.domain.Reservation;
+
+@Controller
+public class ReservationController {
+
+    private final List<Reservation> reservations = new ArrayList<Reservation>();
```

Comment:
```suggestion
    private final List<Reservation> reservations = new ArrayList<>();
```
이렇게 바꾸면 어떤 장점이 있을까요?

Reply:
늘 지적받는 부분인데 아직도 저 부분을 비워두는 게 잘 안되네요… 비워놓아도 자바의 타입 추론을 통해 자료형이 결정되기에  저는 코드의 중복을 피하여 가독성을 높이기 위함이라고 배웠던 것 같아요!

Reply:
추가로 사소한 포인트이지만 유지보수성도 좋아져요! 
List<T> 내부의 T 타입이 현재는 Reservation이에요. 만약 다른 타입으로 변경된다면 new ArrayList<Reservation>();에 정의된 타입도 함께 변경해야 한다는 단점이 있겠죠 😀